docs/using-mina/how-to-delegate.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: How to Delegate
+---
+
+import Subhead from "@site/src/components/common/Subhead";
+
+# How to Delegate
+
+<Subhead>
+Learn how to delegate MINA and receive staking rewards
+</Subhead>
+
+Instead of running your own node, you can delegate MINA to a validator (block producer) and receive staking rewards.
+
+### Prerequisites
+
+First, ensure you installed a Mina wallet with funds.
+
+### Delegating MINA
+
+Here’s how to delegate MINA within common wallets:
+
+- [**Auro**](https://www.aurowallet.com) - Open the wallet, tap “*Staking*”, tap “*Go to Staking*”, select a block producer, and confirm the transaction details.
+
+<center>
+<figure>
+  <img
+    src="/img/Staking_MINA.png"
+    alt="Screenshot of Mina Explorer"
+    width="95%"
+  />
+</figure>
+</center>
+<br></br>
+
+- [**Clorio Wallet**](https://clor.io) -  Open the wallet, tap “*Staking Hub*”, select and confirm your validator, and enter your nonce, transaction fee, and passphrase or private key.
```

Comment:
Does Clorio require them to manually enter an nonce? Most people won't know what to do for this step. Would be good to confirm it's optional and then can just not mention it in the text

Reply:
Nope its not required - I removed it